src/core/net/srp_client.cpp

```diff
@@ -721,6 +730,15 @@ void Client::SendUpdate(void)
 
     VerifyOrExit(message != nullptr, error = kErrorNoBufs);
     SuccessOrExit(error = PrepareUpdateMessage(*message));
+
+    if (message->GetLength() + sizeof(Ip6::Udp::Header) + sizeof(Ip6::Header) >= Ip6::kMaxDatagramLength)
+    {
+        LogInfo("Msg is larger than MTU, enabling single service mode");
```

Comment:
nit: log as well the total message length and MTU length.

Reply:
Added message length to the logged info.